plugins/rh-rbac-backend/src/service/policy-builder.ts

```diff
@@ -35,27 +35,20 @@ export class PolicyBuilder {
     discovery: PluginEndpointDiscovery;
     identity: IdentityApi;
     permissions: PermissionEvaluator;
+    database: PluginDatabaseManager;
   }): Promise<Router> {
-    // TODO: Replace with a DB adapter.
-    const fileAdapter = new FileAdapter(
```

Comment:
Do we want to leave this in as there is an additional issue on making the [file adapter configurable](https://github.com/janus-idp/backstage-showcase/issues/360)?

Reply:
Ok, I turned back this code.